<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>220</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>27</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>220</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>27</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>91</y>
     <width>451</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_name">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 17pt &quot;Monotype Corsiva&quot;;

</string>
        </property>
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_name_2">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_date_res">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 17pt &quot;Monotype Corsiva&quot;;

</string>
        </property>
        <property name="text">
         <string>Дата результатов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date_res_2">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_score">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 17pt &quot;Monotype Corsiva&quot;;

</string>
        </property>
        <property name="text">
         <string>Баллы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_score_2">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_res">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 17pt &quot;Monotype Corsiva&quot;;

</string>
        </property>
        <property name="text">
         <string>Результат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_res_2">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>432</width>
     <height>56</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000000</width>
     <height>10000000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 255);
font: italic 28pt &quot;Monotype Corsiva&quot;;

</string>
   </property>
   <property name="text">
    <string>Подробная информация</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QPushButton" name="edit_button">
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
border: 3px solid #5555ff;
color: blue;
background-color:rgb(255, 170, 0);</string>
      </property>
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="del_button">
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
border: 3px solid #5555ff;
color: blue;
background-color: rgb(255, 60, 1);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="edit_res">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>220</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>27</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_box_score">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>220</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>27</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
